.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,43 @@
+name: CI Experiment
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Not a big deal but there's v3 of `checkout` and `setup-java` too (couple of lines lower).
For `setup-java@v3` java-version needs to be `8` fwtw.